services/openshift/templates/common/jenkins-slave-template.json

```diff
@@ -1,111 +0,0 @@
-
```

Comment:
@praveenkumar Any reason to remove this template? upstream is here https://github.com/openshift/origin/blob/master/examples/jenkins/master-slave/jenkins-slave-template.json